code/modules/food_and_drink/plants.dm

```diff
@@ -120,17 +120,29 @@
 			var/turf/T = get_turf(src) // we might have moved during the sleep, so figure out where we are
 			if (T && !src.pooled)
 				explode(T)
-
+*/
 	throw_impact(var/atom/A)
 		if(src.lit) return //cant be high
 		var/turf/T = get_turf(A)
+		var/mob/living/carbon/human/H = A
+		var/datum/plantgenes/DNA = src.plantgenes
 		if(!T) return
 		if(!T || src.pooled) return
-		explode(T)
-		src.visible_message("<span class='alert'>[src] splats onto the floor explosively!</span>")
+		//explode(T)
+		fireflash(T,1,1)
+		if(istype(H))
+			H.TakeDamage("chest",0,clamp(DNA.potency/2,10,50) + max(DNA.potency/5-20, 0)*(1-H.get_heat_protection()/100),0)//burn damage is half of the potency, soft capped at 50, with a minimum of 10, any extra potency is divided by 5 and added on. The resulting number is then reduced by heat resistance, and applied to the target.
+			H.update_burning(DNA.potency * 2)
```

Comment:
I believe this proc changes burning by seconds, not ticks - meaning a 50 potency tomato would max out the burning status duration. May wish to reconsider the scaling

Reply:
It does? I was under the impression it used deciseconds. I'll change the scaling now.